src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface State {
+  info: string | null;
+}
+
+export class App extends React.Component <{}, State> {
+  state = {
+    info: null,
```

Comment:
As per task description
```suggestion
    keyPressed: null,
```